tests/ci/create-commit.sh

```diff
@@ -7,15 +7,15 @@ ID=${ID:-}
 ARCH=${ARCH:-}
 IMAGE_TYPE=${IMAGE_TYPE:-}
 HTTPD_PATH=${HTTPD_PATH:-}
+SOURCE_FILE=${SOURCE_FILE:-}
+BLUEPRINT_FILE=${BLUEPRINT_FILE:-}
 
 # Set up variables.
 TEST_UUID=$(uuidgen)
 IMAGE_KEY="osbuild-composer-ostree-test-${TEST_UUID}"
 
 # Set up temporary files.
 TEMPDIR=$(mktemp -d)
-BLUEPRINT_FILE=${TEMPDIR}/blueprint.toml
-SOURCE_FILE=${TEMPDIR}/copr_neptune.toml
```

Comment:
maybe `APPLICATION_SOURCE_FILE` or `NEPTUNE_SOURCE_FILE` would be better?

Reply:
Can you please explain the `cloud-init` suggestion? I am not following how that can work and what is the value.